sssd_test_framework/utils/sssd.py

```diff
@@ -768,3 +768,11 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def passkey(self) -> None:
+        """
+        Configure SSSD with passkey authentication.
+
+        #. Set pam_passkey_auth to True
+        """
+        self.sssd.pam["pam_passkey_auth"] = "true"
```

Comment:
You can remove this commit as this isn't needed anymore. By default `pam_passkey_auth` is true.

Reply:
Thanks, I removed it.